articles/css-grid/index.md

```diff
@@ -0,0 +1,379 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/css-flexbox/
+title: Create Two-Dimensional Layouts for Websites using CSS Grid
+description: CSS Grid is a two-dimensional positioning layout system in CSS, that can be used to create responsive interfaces for the web.
+author: saiharsha-balasubramaniam
+date: 2020-09-16T00:00:00-08:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/css-grid/hero.jpg
+    alt: css grid
+---
+
+CSS Grid is a two-dimensional positioning layout system in CSS, that can be used to create responsive interfaces for the web. Similar to Flexbox, CSS rules are applied to the parent container and the children.
+
+CSS Flexbox is a one dimension positioning system. Check out [this](https://www.section.io/engineering-education/css-flexbox/#conclusion) article to learn more about CSS Flexbox.
+
+## Table of Contents
+
+- [Introduction & History](#introduction-&-history)
+- [Terminology](#terminology)
+- [Let's Code](#let's-code)
+- [Further Reading](#further-reading)
+
+## Introduction & History
+
+Using CSS for web layouts has always been a tricky affair. We used tables and floats to position items before. They were inefficient hacks and they weren't intuitive.
+
+Therefore, CSS Flexbox was introduced and it made positioning easier. But as the complexity of your website layout increased, Flexbox wasn't the best option. It was designed for one-dimensional website layouts. Thus, CSS grid solves that problem by introducing features for building two-dimensional layouts.
```

Comment:
You could here or at the end mention the difficulties of creating your final grid example with Flexbox to further highlight the advantages of CSS Grid.